roles/kubernetes/node/templates/kubelet.env.v1beta1.j2

```diff
@@ -38,9 +31,7 @@ KUBELET_ARGS="{{ kubelet_args_base }} {% if node_taints|default([]) %}--register
 {% if kubelet_flexvolumes_plugins_dir is defined %}
 KUBELET_VOLUME_PLUGIN="--volume-plugin-dir={{ kubelet_flexvolumes_plugins_dir }}"
 {% endif %}
-{% if kube_network_plugin is defined and kube_network_plugin in ["calico", "canal", "cni", "flannel", "weave", "cilium", "kube-ovn", "kube-router", "macvlan"] %}
-KUBELET_NETWORK_PLUGIN="--network-plugin=cni --cni-conf-dir=/etc/cni/net.d --cni-bin-dir=/opt/cni/bin"
-{% elif kube_network_plugin is defined and kube_network_plugin == "cloud" %}
```

Comment:
Why can we delete those lines?
I cannot find lines related to Kubernetes versions and dockershim here.

Reply:
Because those parameters affected only the dockershim, they will throw an error in 1.24 and are ignored when using something else than docker with versions less than 1.24. See the Note on https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#installation